physics/PBL/SATMEDMF/satmedmfvdifq.F

```diff
@@ -83,9 +86,19 @@ subroutine satmedmfvdifq_run(im,km,ntrac,ntcw,ntrw,               &
      &     dspheat,dusfc,dvsfc,dtsfc,dqsfc,hpbl,dkt,dku,                &
      &     kinver,xkzm_m,xkzm_h,xkzm_s,dspfac,bl_upfr,bl_dnfr,          &
      &     rlmx,elmx,sfc_rlm,tc_pbl,                                    &
+     &     do_canopy, vegtype, lai,                                     &
+!IVAI: canopy inputs
+     &     claie, cfch, cfrt, cclu, cpopu,
+!IVAI
+!TODO -Canopy Inputs
+!     &     rdcanopylai, rdcanopyfch, rdcanopyfrt, rdcanopyclu,        &
+!     &     canopylaixy, canopyfchxy, canopyfrtxy, canopycluxy,        & 
      &     ntqv,dtend,dtidx,index_of_temperature,index_of_x_wind,       &
      &     index_of_y_wind,index_of_process_pbl,gen_tend,ldiag3d,       &
-     &     errmsg,errflg)
+     &     errmsg,errflg,                                               &
+!IVAI: aux arrays
```

Comment:
These aux arrays are purely for diagnosing the implemention, correct? I believe they need to be commented out or removed before merging this into ufs/dev

Reply:
Yes please, thanks.